File: LeetCode/1304.py
#Link: https://leetcode.com/problems/find-n-unique-integers-sum-up-to-zero/

class Solution:
    def sumZero(self, n: int) -> List[int]:
        array = []
        begin = -(n//2)
        end = n//2
        for i in range(begin, end+1):
            array.append(i)
        if n%2 == 0: array.remove(0)
        return array
        

File: LeetCode/cpp/1304.cpp
//Link: https://leetcode.com/problems/find-n-unique-integers-sum-up-to-zero/

class Solution {
public:
    vector<int> sumZero(int n) {
        vector<int> array;
        int begin = -(n/2);
        int end = n/2;
        for(int i = begin ; i <= end ; i++)
            array.push_back(i);
        if(n % 2 == 0)
            array.erase(array.begin() + n/2);
        return array;
    }
};
